src/main/java/net/openhft/chronicle/core/util/Ints.java

```diff
@@ -32,7 +32,6 @@
  * A collection of functional compositions to check and assert various requirements
  * related to {@code int} values.
  */
-@Deprecated(/* to be moved in x.26 */)
```

Comment:
why has this been undeprecated?

Reply:
A few functions are used many times, but most aren't used at all.